File: Python/Samples/Command/Invoker.py
from Commands import Command

class Invoker:
    def __init__(self):
        self.commands=[]
    
    def addCommand(self, cmd:Command):
        self.commands.append(cmd)

    def cancelCommand(self, cmd:Command):
        self.commands.remove(cmd)

    def invoke(self):
        for cmd in self.commands:
            cmd.execute()

File: CSharp/DP.Domain/Samples/Command/Invoker.cs
using System.Collections.Generic;
using System.Linq;

namespace DP.Domain.Samples.Command
{
    public class Invoker
    {
        private IList<Command> _commands = null;

        public Invoker()
        {
            this._commands = new List<Command>();
        }

        public void AddCommand(Command command)
        {
            this._commands.Add(command);
        }

        public void CancelCommand(Command command)
        {
            this._commands.Remove(command);
        }

        public void Invoke()
        {
            foreach(var cmd in this._commands)
            {
                cmd.Execute();
            }
        }
    }
}